app/src/main/java/edu/mills/cs115/AZCategoryFragment.java

```diff
@@ -18,36 +16,46 @@
 import android.view.ViewGroup;
 import android.widget.Toast;
 
-public class AZCategoryMaterialList extends Fragment {
-    private static final String TAG = "AZCategoryMaterialList";
-    private AZTermList azTermList;
+/**
+ * AZCategoryFragment implements the view for categories within the dictionary database. Categories are
+ * displayed in a recycler view containing card view's.
```

Comment:
* displayed in a recycler view containing card views[card's views? or card views?].